app/src/main/java/com/innowisegroup/datetimepicker/DatePickerFragment.kt

```diff
@@ -125,6 +131,13 @@ class DatePickerFragment : Fragment() {
             Bundle().apply { putString(UPDATE_DATE_TAB_TITLE_KEY, newValue.formatDate()) })
```

Comment:
Do you need here "apply" function?

Also change in 123 line to - "day ?: return"